pkg/connectorrunner/runner.go

```diff
@@ -750,7 +772,12 @@ func NewConnectorRunner(ctx context.Context, c types.ConnectorServer, opts ...Op
 	runner.cw = cw
 
 	if cfg.onDemand {
-		if cfg.c1zPath == "" && cfg.eventFeedConfig == nil && cfg.createTicketConfig == nil && cfg.listTicketSchemasConfig == nil && cfg.getTicketConfig == nil && cfg.bulkCreateTicketConfig == nil {
+		if cfg.c1zPath == "" &&
+			cfg.eventFeedConfig == nil &&
+			cfg.createTicketConfig == nil &&
+			cfg.listTicketSchemasConfig == nil &&
+			cfg.getTicketConfig == nil &&
+			cfg.bulkCreateTicketConfig == nil {
```

Comment:
did we also want to check `cfg. listActionSchemasConfig == nil` here?

Reply:
Fixed in [79a4cfb](https://github.com/ConductorOne/baton-sdk/pull/564/commits/79a4cfbf6153c0de44ca78dc9ccf216dcd5770ab)